lib/hyacinth/jobs/batch_export_job/export_filter.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module Hyacinth
+  module Jobs
+    module BatchExportJob
+      class ExportFilter
+        DEFAULT_CORE_INCLUSION_FILTERS = [
+          '_uid',
+          '_asset_type',
+          '_digital_object_type',
+          '_doi',
+          /_identifiers\[\d+\]/,
+          /_parent_digital_objects\[\d+\]\..+/,
+          '_primary_project.string_key',
```

Comment:
Just a quick note here to say that we'll definitely be updating the DEFAULT_CORE_INCLUSION_FILTERS at some point, but this represents a basic initial set.  We can talk with the Hyacinth User Group to figure out which additional fields would be useful to include in batch exports in the future.